experiments/model_c/api.py

```diff
@@ -0,0 +1,72 @@
+"""
+tts package supports using command line or python to call model for training
+or inferencing. Here we deomnstrate how to use python to call models
+"""
+from typing import Optional, Union
+import sounddevice as sd
+from scipy.io import wavfile
+import torch
+from TTS.api import TTS
+
+
+DeviceLikeType = Union[str, torch.device, int]
+
+
+def get_device() -> str:
+    """Check if cuda is available and return corresp. string"""
+    device = "cuda" if torch.cuda.is_available() else "cpu"
+    return device
+
+
+def get_tts_model(
+    identifier: Optional[str] = None, device: DeviceLikeType = "cpu"
```

Comment:
Input argument `device` is missed in docstr. FYI.

Reply:
solved, thanks!